### DISCUSSION TITLE
Add download option to map viewer

### DISCUSSION THREAD

**Discussion by geoxfinity** [lsc-hubs/kenya-hub#10] [General]
Please a download capability on this page

**Comment by pvgenuchten** [comment]
Thank you for the idea @geoxfinity, you have specific datasets in mind for which you would like a download. Notice that you can download (if download option is available) from the metadata section.

**Reply by pvgenuchten** [reply]
From other comments we noticed there is interest to download a subset of a dataset for an area of interest.

I prepared a tutorial on how to extract a subset using qgis at https://lsc-hubs.github.io/kenya-hub/docs/user/ows-in-qgis.html

Would be interesting to hear from you if you can proceed for now?
